src/videojs-http-streaming.js

```diff
@@ -325,7 +325,8 @@ class HlsHandler extends Component {
             videojs.log.warn('player.hls is deprecated. Use player.tech().hls instead.');
             tech.trigger({ type: 'usage', name: 'hls-player-access' });
             return this;
-          }
+          },
+          configurable: true
```

Comment:
What's this property for?

Reply:
By default configurable is false, means you can't change the value once set.

Reply:
change including delete.